### DISCUSSION TITLE
Dependency + configuration

### DISCUSSION THREAD

**Discussion by stepango** [formatools/forma#69] [Ideas]
what if need dependency and we need to make sure it always will be `compileOnly`

so potentially we can do smth like
```kotlin
val someDep = "some.dep:ver:1.0".dep(CompileOnly)
```

Another example 

```kotlin
val testDep = "some.dep:ver:2.0.dep(TestImplementation, AndroidTestImplementation)
```

**Comment by michaem** [comment]
I think it good, bro :)
And for parameterless .dep will be implementation as default?
